setup.py

```diff
@@ -510,7 +489,6 @@ def dosetup(name, version, packages, datafiles, scripts, ext_modules=[]):
         scripts=scripts,
         ext_modules=ext_modules,
         cmdclass=cmdclass,
-        install_requires=parse_requirements('requirements/required.txt'),
```

Comment:
This probably should be kept but instead point to `recommended.txt` like https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies @nijel thoughts on this?

Reply:
I see that this is kinda proposed in #4004